src/WRS.jl

```diff
@@ -56,5 +57,14 @@ function pb2gen(x, y, est, q=0.5; alpha=0.05, nboot=2000)
     return est_x, est_y, est_diff, ci, pvalue, se
 end
 
+function to_be_named(x, y, est, quantiles=[0.05, 0.25, 0.5, 0.75, 0.95]; alpha=0.05, nboot=2000)
```

Comment:
Do we want this default list of quantiles?

Reply:
Yes, I like a default list.